src/win/util.c

```diff
@@ -1624,6 +1624,10 @@ int uv_os_uname(uv_utsname_t* buffer) {
      https://github.com/gagern/gnulib/blob/master/lib/uname.c */
   OSVERSIONINFOW os_info;
   SYSTEM_INFO system_info;
+  HKEY registry_key;
+  WCHAR product_name_w[256];
+  LPDWORD product_name_w_size;
```

Comment:
```suggestion
  DWORD product_name_w_size;
```
LP is for Long Pointer. This should be just a integer.

Reply:
The API docs call for an LPDWORD though?

Reply:
You take the address of it with `&`

Reply:
It's an in/out

Reply:
Oh. Yea I fixed that on the previous push.